seqr/models.py

```diff
@@ -1104,6 +1104,35 @@ class Meta:
         json_fields = ['gene_id', 'tpm']
 
 
+class RnaSeqSpliceOutlier(DeletableSampleMetadataModel):
+    STRAND_CHOICES = (
+        ('+', '5′ to 3′ direction'),
+        ('-', '3′ to 5′ direction'),
+        ('*', 'Any direction'),
+    )
+
+    p_value = models.FloatField()
+    z_score = models.FloatField()
+    chrom = models.CharField(max_length=2)
+    start = models.IntegerField()
+    end = models.IntegerField()
+    strand = models.CharField(max_length=1, choices=STRAND_CHOICES)  # "+", "-", or "*"
+    gene_name = models.CharField(max_length=20)
+    type = models.CharField(max_length=12)
+    delta_psi = models.FloatField()
+    read_count = models.IntegerField()  # RNA-seq reads that span the splice junction
+    dot_size = models.FloatField()
+    rare_disease_samples_with_junction = models.IntegerField()
+    rare_disease_samples_total = models.IntegerField()
```

Comment:
wre should only be stroing fields that need displaying - I think it would be helpful to update the display ticket to be more explicit about what we will actually be needing